CHANGELOG.md

```diff
@@ -148,12 +147,12 @@ v0.33.0 (2023-04-25)
   compiler issues are resolved and 32-bit ARM builds are stable. (@rfratto)
 
 - Agent Management: `agent_management.api_url` config field has been replaced by
-`agent_management.host`. The API path and version is now defined by the Agent. (@jcreixell)
+  `agent_management.host`. The API path and version is now defined by the Agent. (@jcreixell)
 
 - Agent Management: `agent_management.protocol` config field now allows defining "http" and "https" explicitly. Previously, "http" was previously used for both, with the actual protocol used inferred from the api url, which led to confusion. When upgrading, make sure to set to "https" when replacing `api_url` with `host`. (@jcreixell)
 
 - Agent Management: `agent_management.remote_config_cache_location` config field has been replaced by
-`agent_management.remote_configuration.cache_location`. (@jcreixell)
+  `agent_management.remote_configuration.cache_location`. (@jcreixell)
```

Comment:
Q: are these formatting changes intentional ? (just asking in case they're not)

Reply:
Oh no they are not. This is cause by the auto formatter of my editor.